ClientApp/src/app/flights-search-bar/flights-search-bar.component.css

```diff
@@ -0,0 +1,66 @@
+.main_container{
+  margin: auto;
+  padding-top: 1em;
+  width: 30em;
+  padding-bottom: 1em;
+  display: flex;
+  flex-direction: column;
+}
+
+.input_container{
+  margin: auto;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.direction_field {
+  width: 40%;
+}
+
+.mat-radio-button ~ .mat-radio-button {
+  margin-left: 15px;
+}
+
+:host ::ng-deep .mat-radio-container{
+  height: 12px;
+  width: 12px;
+}
+:host ::ng-deep .mat-radio-outer-circle{
+  height: 12px;
+  width: 12px;
+}
+:host ::ng-deep .mat-radio-inner-circle{
+  height: 12px;
+  width: 12px;
+}
+:host ::ng-deep .mat-radio-button .mat-radio-ripple{
+  height: 20px; /*double of your required circle radius*/
+  width: 20px;  /*double of your required circle radius*/
+  left: calc(50% - 10px); /*'10px'-same as your required circle radius*/
+  top: calc(50% - 10px); /*'10px'-same as your required circle radius*/
```

Comment:
Comments are not allowed on master

Reply:
Correction - obvious or meaningless comments shouldn't be left in the code. Same goes for commented out pieces of code. 
In this particular case not sure if information here adds any value to the code but I leave it to you to decide.